plugins/woocommerce/src/Internal/Admin/Homescreen.php

```diff
@@ -117,6 +117,8 @@ public function maybe_set_default_shipping_options_on_home( $settings ) {
 		if (
 			( 'US' === $country_code && $is_jetpack_installed && $is_wcs_installed && $is_jetpack_connected )
```

Comment:
@moon0326 Can we remove this check?  It seems like line 120 is enough.

Reply:
@chihsuan I think they're different 🤔 . The new addition covers an edge case where Jetpack is installed, but WCS is not. `( 'US' === $country_code && $is_jetpack_installed && $is_wcs_installed && $is_jetpack_connected )` only hit when both WCS and Jetpack are installed and Jetpack is connected.

Reply:
Yeah, they're different. But when `( 'US' === $country_code && $is_jetpack_installed && $is_wcs_installed && $is_jetpack_connected )` hit, then `'US' === $country_code && $is_jetpack_installed` must be true. Since we use `||` operator, we can just check `'US' === $country_code && $is_jetpack_installed`? Did I miss anything? 🤔

Reply:
@chihsuan That's true, but we still need both to cover an edge case.

Let's say there is a user who goes through the OBW process and install both Jetpack and WCS then approve the Jetpack connection. The user hits the first `( 'US' === $country_code && $is_jetpack_installed && $is_wcs_installed && $is_jetpack_connected )` and then process the if block.

Let's say another user goes through the OBW process and only installs Jetpack.

The user will not satisfy `( 'US' === $country_code && $is_jetpack_installed && $is_wcs_installed && $is_jetpack_connected )`, but hits ( 'US' === $country_code && $is_jetpack_installed ).

Perhaps, you want to change the 2nd condition to `( 'US' === $country_code && $is_jetpack_installed && false == $is_wcs_installed)` to make it a bit clear?

Reply:
@moon0326 Thanks for clarifying!

> The user will not satisfy ( 'US' === $country_code && $is_jetpack_installed && $is_wcs_installed && $is_jetpack_connected ), but hits ( 'US' === $country_code && $is_jetpack_installed ) instead.

I think the user will satisfy `( 'US' === $country_code && $is_jetpack_installed )` in both cases? So I was thinking we just need the new addition.

> Perhaps, you want to change the 2nd condition to ( 'US' === $country_code && $is_jetpack_installed && false == $is_wcs_installed) to make it a bit clear?

This looks good to me as well, but do we need to cover an edge case that a user installed jetpack and WCS but the jetpack is not connected?

Reply:
> I think the user will satisfy ( 'US' === $country_code && $is_jetpack_installed ) in both cases? So I was thinking we just need the new addition.

I think I finally understood what you mean. I'm sorry it took me some time to understand. I'm still a bit confused with all these conditions 🥲 . I guess we don't need to check WCS after all...in that case?

Reply:
@chihsuan Removed the first check-in https://github.com/woocommerce/woocommerce/pull/33788/commits/54baff1abef6c9f36ae9eb5bb9b9dba7c5c4f68e

cc @ilyasfoo 

@ilyasfoo Could you also take a look at the conditions to make sure I'm covering all the requirements please?

Reply:
Sorry, I just arrived at this ping. Will look at it now

Reply:
Thank you for the review @chihsuan  @ilyasfoo  👍  Let's go with the current change then.